src/modules/auth.js

```diff
@@ -33,6 +32,32 @@ const login = async (event, dispatch, history) => {
   }
 };
 
+const signUp = async (event, dispatch, history) => {
+  event.preventDefault();
+  try {
+    const email = event.target.email.value;
+    const password = event.target.password.value;
```

Comment:
I agree with Mauro here, you should send the password confirmation. If you are uncertain, then read the documentation for device token auth

Reply:
yes mauro explained to us that it is different for sign in and sign up, so now i understand! 
Now we send it!